time stamp_obs_update
bool[] real_obs_received
bool[] obs_with_noise
string task_status # success, fail or ongoing
string who_succeeded
